src/pages/demo.vue

```diff
@@ -303,23 +313,89 @@
     description="Text"
     :close="true"
     ref="bottomSheet"
+    controlType="normal"
+    :showPrimaryButton="true"
+    primaryButtonLabel="Primary !"
+    :showSecondaryButton="true"
+    secondaryButtonLabel="Secondary !"
+    @primaryButtonClick="onClickPrimary"
+    @secondaryButtonClick="onClickSecondary"
```

Comment:
just tip to check for each every method/computed proped defined in script ion console s